subsys/bluetooth/mesh/access.c

```diff
@@ -1566,6 +1569,19 @@ int bt_mesh_model_recv(struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *buf)
 	return err;
 }
 
+int bt_mesh_access_recv(struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *buf)
```

Comment:
Could you please add to the commit message that this function was introduced to cover the case with opcode aggregator messages? So that we remember why we did this.